### PR TITLE
Fixed double quotes issue in powershell_out

### PR DESCRIPTION
**Pull request by NimishaS**
Fix for https://github.com/chef/chef/issues/5049

Re-worked changes as suggested by @mwrock : https://github.com/chef/chef/pull/5420

### CODE REVIEW

**Comment by NimishaS** [comment]
@btm , @mwrock please review this

File: spec/unit/mixin/powershell_out_spec.rb [hunk 1]
```diff
@@ -44,6 +44,17 @@
       ).and_return(ret)
       expect(object.powershell_out("Get-Process", timeout: 600)).to eql(ret)
     end
+
+    context "when double quote is passed in the powershell command" do
+      it "passes if double quote is appended with single escape" do
+        result = object.powershell_out("Write-Verbose \"Some String\" -Verbose")
+        expect(result.stderr).to be == ""
```
Comment (mwrock) [inline spec/unit/mixin/powershell_out_spec.rb]: Not a blocker, but you might as well test stdout to make sure the command emits the expected outpit to be 100% sure the command ran as expected.

Comment (NimishaS) [inline spec/unit/mixin/powershell_out_spec.rb]: Done @mwrock

**Comment by NimishaS** [comment]
@mwrock please merge this if this looks ok.

**Pull request merged**